src/features/basic/slider-improvements/slider-improvements.ts

```diff
@@ -0,0 +1,130 @@
+        const column = sliderColumns.get(columnIndex);
+        if (!column) {
+          sliderColumns.set(columnIndex, [slider]);
+        } else {
+          column.push(slider);
+        }
+      });
+      for (const [columnIndex, sliders] of sliderColumns.entries()) {
+        const header = headerRow.children[columnIndex];
+        const minSliderColumn = () => {
+          sliders.forEach(slider => {
+            const markContainer = _$(slider, 'rc-slider-mark');
+            if (markContainer && !slider.classList.contains('rc-slider-disabled')) {
+              clickElement(markContainer.firstElementChild as HTMLElement);
+            }
+          });
+        };
+        const maxSliderColumn = () => {
+          sliders.forEach(slider => {
+            const markContainer = _$(slider, 'rc-slider-mark');
+            if (markContainer && !slider.classList.contains('rc-slider-disabled')) {
+              clickElement(markContainer.lastElementChild as HTMLElement);
+            }
+          });
+        };
+        const disabled = sliders.every(slider => slider.classList.contains('rc-slider-disabled'));
+        header.classList.add($style.sliderColumnHeader);
+        createFragmentApp(
+          SliderButton,
+          reactive({
+            onClick: minSliderColumn,
+            label: '<',
+            disabled,
+          }),
+        ).prependTo(header);
+        createFragmentApp(
+          SliderButton,
+          reactive({
+            onClick: maxSliderColumn,
+            label: '>',
+            disabled,
+          }),
+        ).appendTo(header);
+      }
+    }
+  });
+}
+
+function sliderMinMaxButtons(tile: PrunTile) {
+  subscribe($$(tile.anchor, 'rc-slider'), slider => {
+    slider.parentElement?.classList.add($style.sliderView);
+    const disabled = slider.classList.contains('rc-slider-disabled');
+    const markContainer = _$(slider, 'rc-slider-mark');
+    const minSlider = () => clickElement(markContainer?.firstElementChild as HTMLElement);
+    const maxSlider = () => clickElement(markContainer?.lastElementChild as HTMLElement);
+    createFragmentApp(
+      SliderButton,
+      reactive({
+        onClick: minSlider,
+        label: '<',
+        disabled,
+      }),
+    ).before(slider);
+    createFragmentApp(
+      SliderButton,
+      reactive({
+        onClick: maxSlider,
+        label: '>',
+        disabled,
+      }),
+    ).after(slider);
+  });
+}
+
+// ONLY FOR POPID(?)
+function disableInvalidSliders(tile: PrunTile) {
+  subscribe($$(tile.anchor, 'table'), table => {
+    subscribe($$(table, 'rc-slider'), slider => {
+      const sliderMaxMark = _$(slider, 'rc-slider-mark')?.lastElementChild;
+      if (!sliderMaxMark) return;
+      const sliderMax = parseFloat(sliderMaxMark.textContent);
+      const sliderValueMark = _$$(slider, 'rc-slider-mark-text-active').findLast(() => true); // the last active mark-text will be either the min/max or the current value
```

Comment:
Code style for this project: put comments on a separate line, start with a capital letter, and end with a full stop.
```suggestion
      // The last active mark-text will be either the min/max or the current value.
      const sliderValueMark = _$$(slider, 'rc-slider-mark-text-active').findLast(() => true);
```